tests/network/networkpolicy.go

```diff
@@ -388,10 +388,10 @@ func checkHTTPPingAndStopOnFailure(fromVmi, toVmi *v1.VirtualMachineInstance, po
 }
 
 func checkHTTPPing(vmi *v1.VirtualMachineInstance, ip string, port int) error {
-	const curlCheckCmd = "curl --head %s --connect-timeout 5\n"
+	const wgetCheckCmd = "wget -S --spider %s -T 5\n"
```

Comment:
wget adds way more noise than curl, are we ok with this change?

```
[phoracek@t490s ~]$ wget -S --spider google.com -T 5 2>&1 | wc -l
37

[phoracek@t490s ~]$ curl --head google.com --connect-timeout 5 2>&1 | wc -l
14
```

Reply:
alpine doesn't have `curl` so that's the best option we have. Why should we care about this "noise"?

Reply:
well, unfortunately I don't see any other option with alpine.

Reply:
All good then, feel free to cancel the hold any time